### PR TITLE
#212 added shortening URLs

### PR DESCRIPTION
**Pull request by asaen**
#212 Added shortening URLs to the documents

### CODE REVIEW

**Comment by karato** [comment]
@darkled Many thanks for the PR, let me find a reviewer for it

**Comment by karato** [comment]
@caarlos0 do this review

**Comment by caarlos0** [comment]
@rultor good to merge

File: src/main/java/com/nerodesk/om/mock/MkDoc.java [hunk 1]
```diff
@@ -137,6 +138,17 @@ public void write(final InputStream input) throws IOException {
         Logger.info(this, "%s saved", file);
     }
 
+    @Override
+    public String shortUrl() {
+        String url = "No URL";
+        try {
+            url = this.file().toURI().toURL().toString();
+        } catch (final MalformedURLException ex) {
+            Logger.error(this, "Error happened: %s", ex.getMessage());
```
Comment (yegor256) [inline src/main/java/com/nerodesk/om/mock/MkDoc.java]: hm.. are you serious? swallowing the exception? see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#exceptions

**Comment by yegor256** [comment]
@darkled one comment above

**Comment by asaen** [comment]
@yegor256 yes, you're right, thank you.
@caarlos0 @yegor256 please take a look once more

**Comment by yegor256** [comment]
@rultor merge

**Comment by caarlos0** [comment]
@darkled you have merge conflicts

**Comment by asaen** [comment]
@caarlos0 @yegor256 please try to merge once more if alright

**Comment by yegor256** [comment]
@rultor merge

**Comment by karato** [comment]
@caarlos0 **17 mins** was added to your account, many thanks for your contribution! The task took AP-9HA088315W067635W.... you're getting extra minutes here (c=2)... +17 to your rating, your total score is [+2007](http://www.netbout.com/b/35095?open=rating)

**Comment by karato** [comment]
@rultor deploy pls

**Pull request merged**